LambdaEngine/Source/Debug/CPUProfiler.cpp

```diff
@@ -105,7 +105,7 @@ namespace LambdaEngine
 	void CPUProfiler::ToggleSample(EKey key, uint32_t frameCount)
```

Comment:
Seems like you have a LINT warning here

Reply:
While the function can be static as the linter says, this would create a confusing interface for the class, since there is a Get() function in order to access the functions. Having one static function while the rest of the interface isn't would be confusing.